modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -487,6 +487,167 @@ func (suite *KeeperTestSuite) TestChanUpgradeAck() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestChanUpgradeOpen() {
+	var path *ibctesting.Path
+	testCases := []struct {
+		name     string
+		malleate func()
+		expError error
+	}{
+		{
+			"success, counterparty in TRYUPGRADE",
+			func() {
+				err := path.EndpointA.ChanUpgradeInit()
+				suite.Require().NoError(err)
+
+				err = path.EndpointB.ChanUpgradeTry()
+				suite.Require().NoError(err)
+
+				err = path.EndpointA.ChanUpgradeAck()
+				suite.Require().NoError(err)
+
+				channelB := path.EndpointB.GetChannel()
+				channelB.FlushStatus = types.FLUSHCOMPLETE
+				path.EndpointB.SetChannel(channelB)
```

Comment:
I think once #3928 is implemented, this won't be necessary, should we add a note?